setup/files/supervisord.conf

```diff
@@ -5,13 +5,13 @@ pidfile=/opt/redash/supervisord/supervisord.pid
 directory=/opt/redash/current
 
 [inet_http_server]
-port = 127.0.0.1:9001
+port = 0.0.0.0:9001
```

Comment:
It's better from security perspective. The Python Web server is more
vulnerable than nginx that proxies it.

But anyway better not to have the supervisord control port exposed because
iirc it has no authentication.
On Oct 6, 2015 8:38 PM, "John Wu" <notifications@github.com> wrote:

> In setup/files/supervisord.conf
> <https://github.com/EverythingMe/redash/pull/588#discussion_r41296346>:
>
> > @@ -5,13 +5,13 @@ pidfile=/opt/redash/supervisord/supervisord.pid
> >  directory=/opt/redash/current
> >
> >  [inet_http_server]
> > -port = 127.0.0.1:9001
> > +port = 0.0.0.0:9001
>
> Well, I think it might be (generally) a better practice to bind to 0.0.0.0.
> I mean, there is no reason not to do so :) Do you have further thinking on
> this?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/EverythingMe/redash/pull/588/files#r41296346>.
>